src/Locale.php

```diff
@@ -36,6 +37,7 @@ public function __construct(
         private TranslatorInterface $translator,
         private UrlGeneratorInterface $urlGenerator,
         private SessionInterface $session,
+        private Aliases $aliases,
```

Comment:
Suggest create method `Locale::withLocationPrefix()` and call him in `SubFolder`. It's allow don't add new dependency to this middleware.

Reply:
It's yii-specific package, so I don't think it's needed.

Reply:
In Yii app user can use another alias instead of `@baseUrl`.

Reply:
Now added aliases dependency, option `$baseUrlAliases` and method `withBaseUrlAlias()`. Instead it we can add only option `$locationPrefix` and method `withLocationPrefix()`. Seems, It will be easier and don't need add new dependency.